site/content/docs/manifest/rd-web-service.en.md

```diff
@@ -137,6 +139,22 @@ Amount of memory in MiB reserved for each instance of your service. See the [AWS
 
 <div class="separator"></div>
 
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>      
+The `network` section contains parameters for connecting the service to AWS resources in the environment's VPC.  
+By connecting the service to a VPC, you can use [service discovery](../developing/service-discovery.en.md) to communicate with other services
+in your environment, or connect to a database in your VPC with [`storage init`](../commands/storage-init.en.md).
+
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>    
+Subnets in the VPC to route egress traffic from the service.
+
+<span class="parent-field">network.vpc.</span><a id="network-vpc-placement" href="#network-vpc-placement" class="field">`placement`</a> <span class="type">String</span>  
+Must be `'private'` for the App Runner service to route egress traffic through the private subnets of the VPC.  
```

Comment:
Just wondering if we need to clarify that this is the only option; that if they don't want `private` they should leave it commented out...?